security/tools/jwt/samples/gen-jwt.py

```diff
@@ -28,9 +28,10 @@
 def main(args):
     """Generates a signed JSON Web Token from local private key."""
     with open(args.key) as f:
-        pem_data = f.read()
+        pem_data_encode = f.read()
     f.closed
 
+    pem_data = pem_data.encode("utf-8")
```

Comment:
will this break python 2.x now? Not sure if this is a concern though

Reply:
I am test with Python 2.7.16 is ok.But I am not sure about the previous 2.x version, sorry.

Reply:
There are some variable name errors that have been modified.
latest：
```
"""Generates a signed JSON Web Token from local private key."""
    with open(args.key) as f:
        pem_data = f.read()
    f.closed

    pem_data_encode = pem_data.encode("utf-8")
    key = jwk.JWK.from_pem(pem_data_encode)
```